template/.eslintrc.js

```diff
@@ -0,0 +1,35 @@
+module.exports = {
+    root: true,
+    // extends: ['plugin:echobind/react-native'],
+    extends: [
+      '@react-native-community',
+      'plugin:import/errors',
+      'plugin:import/warnings',
+      'plugin:import/typescript',
+    ],
+    rules: {
+      // enable i18n first
+      // 'react/jsx-no-literals': ['error', { noStrings: true }],
+  
+      // remove below after EB eslint plugin is updated
+      // https://github.com/echobind/eslint-plugin-echobind/pull/3
```

Comment:
I think we can follow this comment and remove everything from line 13 on: https://github.com/echobind/eslint-plugin-echobind/pull/3/files

Reply:
Tried and that works!